lib/liquider/markup_helper.rb

```diff
@@ -1,6 +1,6 @@
 module Liquider::MarkupHelper
-  def tokenize(source)
-    Scanner.new(TextStream.new(source, mode: :liquid)).to_enum
+  def tokenize(source, mode: :text)
+    Liquider::Scanner.new(Liquider::TextStream.new(source), mode: mode).to_enum
```

Comment:
Yes, apparently it uses the scope of whatever object it's included in, not the lexical scope, probably since it's a method, y'know